src/main/java/core/dao/BetDaoImp.java

```diff
@@ -1,9 +1,11 @@
 package core.dao;
 
 import core.db.Storage;
+import core.lib.Dao;
 import core.model.Bet;
 import java.util.List;
 
+@Dao
```

Comment:
IMHO @Dao annotation must be above ConsoleHandler class

Reply:
we should write @dao annotation above class what we want to be injected